File: JS/Car.js
export default class Car {
  constructor(license, driver){
    this.id
    this.license = license
    this.driver = driver
    this._passenger
  }

  get passenger() {
    return this._passenger
  }

  set passenger(passengers) {
    if (passengers != 4) {
      console.log("Debes tener 4 pasajeros")
    } else {
      this._passenger = passengers
    }
  }

  printDataCar() {
    console.group(this.id)
    console.log('License: ' + this.license)
    console.log('Nombre: ' + this.driver.name)
    console.log('DNI: ' + this.driver.document)
    console.log('Pasajeros: ' + this.passenger)
    console.groupEnd()
  }
}
// Para hacerlo con require
// module.exports = Car

File: Python/car.py
from account import Account

class Car:
  id = int
  license = str
  driver = Account("","")
  __passenger = int
  
  def __init__(self, license, driver):
    self.license = license
    self.driver = driver
  
  @property
  def passenger(self):
    return self.__passenger

  @passenger.setter
  def passenger(self, passengers):
    if passengers != 4:
      print("Debes tener 4 pasajeros")
    else:
      self.__passenger = passengers

  def printDataCar(self):
    print("Licencia: " + self.license)
    print("Nombre: " + self.driver.name)
    print("Documento: " + self.driver.document)
    print("Pasajeros: " + str(self.passenger))